nidm/experiment/tools/BIDSMRI2NIDM.py

```diff
@@ -91,7 +91,7 @@ def main(argv):
         if args.jsonld:
             f.write(project.serializeJSONLD())
         else:
-            f.write(project.serializeTurtle())
+            f.write(project.serializeTurtle().decode('ASCII'))
```

Comment:
pynidm is py3 only. why would this be necessary. were you testing on python 3 or 2?

Reply:
I was just **moving** the `decode('ASCI')` away -- not introducing it. I am not sure why one wants this, but it certainly is detrimental for JSON-LD. If you are confident that it being there at all (and not just in the way of JSON-LD) is the real bug, I have no objections to wipe it out entirely. And no, this is all PY3.

Reply:
thanks - will take a look